pkg/reconciler/kubernetes/tektonchain/tektonchain.go

```diff
@@ -680,44 +677,33 @@ func AddControllerEnv(controllerEnvs []corev1.EnvVar) mf.Transformer {
 
 func GenerateSigningSecrets(ctx context.Context) map[string][]byte {
 	logger := logging.FromContext(ctx)
-	// Generate ECDSA key pair
-	privateKey, err := ecdsa.GenerateKey(elliptic.P256(), rand.Reader)
-	if err != nil {
-		logger.Error("Error generating private key:", err)
-		return nil
-	}
 
-	// Convert private key to PKCS8
-	privateKeyBytes, err := x509.MarshalPKCS8PrivateKey(privateKey)
+	keys, err := cosign.GenerateKeyPair(nil)
```

Comment:
can you add a comment, why we use nil, (we dont input a password for private key encryption because it will need a user secret)

Reply:
Could we generate a random password on the user's behalf?